### PR TITLE
Clean up MySQL connector deprecation warning

### PR DESCRIPTION
**Pull request by aednichols**
Also checked whether any of the comments about MySQL were actionable and cleaned them up.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -312,14 +312,6 @@ object Dependencies {
...
   private val dbmsDependencies = List(
     "org.hsqldb" % "hsqldb" % hsqldbV,
-    /*
-    When going to 6.0.x, will need to change the jdbc driver to com.mysql.cj.jdbc.Driver
-    - https://dev.mysql.com/doc/connector-j/6.0/en/connector-j-api-changes.html
-
-    The url may also need the parameters:
-    - serverTimezone=UTC via http://stackoverflow.com/a/36793896/3320205
-    - nullNamePatternMatchesAll=true via https://liquibase.jira.com/browse/CORE-2723
-     */
```
Comment (cjllanwarne) [inline project/Dependencies.scala]: I guess we didn't need these parameters after all?

Comment (aednichols) [reply project/Dependencies.scala]: As far as I could tell from my research (Travis might prove me wrong)

**Comment by kshakir** [comment]
You will likely need two `v37` confs.  During the upgrade tests, new confs are needed to run against the cromwell-37 jar.

Something like these should work:

#### local_37_application.conf
```hocon
include "local_application.conf"
database.db.driver = "com.mysql.jdbc.Driver"
```

#### papi_v2_37_application.conf
```hocon
include "papi_v2_application.conf"
database.db.driver = "com.mysql.jdbc.Driver"
```

**Comment by aednichols** [comment]
Thanks @kshakir, you really saved my bacon on this one. Would have taken me a while to figure out I actually need to add files, instead of just editing existing ones.

**Pull request merged**